packages/backend/src/core/discovery/UpdateMonitor.ts

```diff
@@ -93,13 +94,14 @@ export class UpdateMonitor {
     blockNumber: number,
     timestamp: UnixTime,
   ) {
+    const previousDiscovery = await this.getPreviousDiscovery(projectConfig)
+
     const discovery = await this.discoveryRunner.run(projectConfig, blockNumber)
     this.cachedDiscovery.set(projectConfig.name, discovery)
 
-    const diff = await this.findChanges(
-      projectConfig.name,
-      discovery,
-      projectConfig.hash,
+    const diff = diffDiscovery(
```

Comment:
add logs for which discovery is used for diff:
- committed
- db
- new version

Reply:
info level